src/pages/Dashboard/index.tsx

```diff
@@ -1,19 +1,72 @@
 /** @jsxImportSource @emotion/react */
-import React from 'react';
+import React, { useMemo } from 'react';
+
+import { useIsLgDown, useIsSmDown } from 'hooks/responsive';
+import { Tabs } from 'components';
+import { useTranslation } from 'translation';
 import MyAccount from './MyAccount';
+import MintRepayVai from './MintRepayVai';
 import { SupplyMarket, BorrowMarket } from './Markets';
 import { useStyles } from './styles';
 
 const DashboardUi: React.FC = () => {
+  const { t } = useTranslation();
+  const isLgDown = useIsLgDown();
+  const isSmDown = useIsSmDown();
   const styles = useStyles();
+
+  const tabsContent = useMemo(
+    () => [
+      {
+        title: t('dashboard.markets.tabSupply'),
+        content: <SupplyMarket css={styles.item} />,
+      },
+      {
+        title: t('dashboard.markets.tabBorrow'),
+        content: <BorrowMarket css={styles.item} />,
+      },
+    ],
+    [],
+  );
+
+  const renderMarkets = () => {
+    if (isSmDown) {
+      return (
+        <div css={styles.tabsWrapper}>
+          <h4 css={[styles.tabsHeader, styles.tabsTitle]}>{t('dashboard.markets.title')}</h4>
+          <Tabs css={styles.tabsHeader} tabsContent={tabsContent} />
+        </div>
+      );
+    }
+
+    if (isLgDown) {
```

Comment:
Was there a reason that prevented us from doing this in css?

Reply:
more clear responsive cases

Reply:
there are 3 different cases for mobile/tablets/desktop with different layouts

Reply:
also refactored a bit